docs/root/configuration/upstream/cluster_manager/cds.rst

```diff
@@ -18,3 +18,40 @@ Statistics
 ----------
 
 CDS has a :ref:`statistics <subscription_statistics>` tree rooted at *cluster_manager.cds.*
+
+On-demand CDS
```

Comment:
@markdroth FYI, in case you have some review bandwidth cycles to give this a pass similar to the VHDS one.